sass/components/_badge.scss

```diff
@@ -13,15 +13,23 @@
         line-height: 18px;
     }
 
+    .team-sidebar,
+    .sidebar--left,
+    #SidebarContainer {
+        .active .badge,
+        .badge {
+            background: var(--mention-bg);
+            color: var(--mention-color);
```

Comment:
Is there an rgb variable we can use here?

Reply:
A little while back, we decided to have two color variables, hex and rgb. We use rgba() function with `*-rbg` variables when a color needs transparency, otherwise we use the direct hex colour.

I'm 0/5 on whether moving forward we exclusively use rgb() and rbga() function and forgo hex colours, but we should discuss.

@hmhealey, thoughts?

Reply:
I'm 0/5 on one way or another, but we'd have to make sure to provide plugins a timeline to stop using either CSS variable if we were to stop using one ourselves. We'll have to keep supporting hex values in the theme variables on the JS-side since that's what custom themes store in the DB and what any component using the theme utilities will expect, but we don't necessarily need to expose that in the CSS.

The only advantage I see to using two different variables is that it's slightly shorter when we're using full opacity colours, but I'm starting to realize we seem to really like using different opacities a lot 😛

Reply:
@nevyangelova, I'm going to do a combined pass to convert all hex variables uses to rgb in a separate PR, so will leave these as is for now.